src/main/java/net/snowflake/ingest/streaming/internal/BlobBuilder.java

```diff
@@ -48,6 +56,106 @@ class BlobBuilder {
   private static final ObjectMapper MAPPER = new ObjectMapper();
   private static final Logging logger = new Logging(BlobBuilder.class);
 
+  /**
+   * Builds blob.
+   *
+   * @param filePath Path of the destination file in cloud storage
+   * @param blobData All the data for one blob. Assumes that all ChannelData in the inner List
+   *     belongs to the same table. Will error if this is not the case
+   * @param bdecVersion version of blob
+   * @return {@link Blob} data
+   */
+  static <T> Blob build(
```

Comment:
nit: `build` and `buildBlobBytes` might be confusing, should we say something like `constructBlobAndMetadata` and `buildBlob`?